deploy/kubernetes/examples/demo-statefulset-raw-block.yml

```diff
@@ -0,0 +1,35 @@
+#############$$$$$$$$############################
+# StatefulSet example for IBM Block CSI Driver.
+#################################################
+
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: demo-statefulset-raw-block
+spec:
+  selector:
+    matchLabels:
+      app: demo-statefulset
+  serviceName: demo-statefulset
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: demo-statefulset
+    spec:
+      containers:
+      - name: container-demo
+        image: registry.access.redhat.com/ubi8/ubi:latest
+        command: [ "/bin/sh", "-c", "--" ]
+        args: [ "while true; do sleep 30; done;" ]
+        volumeDevices:
+          - name: demo-volume
+            devicePath: "/dev/block"
+      volumes:
+      - name: demo-volume
+        persistentVolumeClaim:
+          claimName: demo-pvc-raw-block
+
+#      nodeSelector:
+#        kubernetes.io/hostname: NODESELECTOR
```

Comment:
I think `NODESELECTOR` should be something like `HOSTNAME`

Reply:
seems like it has been fixed in https://github.com/IBM/ibm-block-csi-driver/pull/167/